### PR TITLE
Adding more information for EBI submission

### PR DESCRIPTION
**Pull request by antgonza**

### CODE REVIEW

**Comment by ackermag** [comment]
This is sufficient.  Only caveat is that right now they are emailing people to tell them it will be public in 10-14 days not 2.

**Comment by antgonza** [comment]
K - changed the ETA to up to 15 days.

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+EBI submission via Qiita
+========================
+
+Qiita allows users to deposit their study, sample, experiment and sequence data to the
+`European Nucleotide Archive (ENA) <https://www.ebi.ac.uk/ena>`__, which is a permanent repository
+part of the `European Bioinformatics Institute (EBI) <https://www.ebi.ac.uk/>`__. Submitting to
+this repository will provide you with a unique identifier for your study, which is generally a
+requirement for publication.
+
+EBI/ENA requires a give set of column fields to describe your samples and experiments, for more
```
Comment (josenavas) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: `a give set` -> `a given set`

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Note that this kind of submissions are time intensive and need full collaboration from the user.
+Thus, do not wait until the last minute to request help. In general, the best time to request a submission
+is when you are writing your paper. Remember that the data can be submitted to EBI and can be
+kept private and simply make public when the paper is accepted. Note that EBI/ENA takes up to 15 days to
+change the status from private to public, so consider this when submitting data and your manuscript.
+
+For convenience Qiita allows you to upload a QIIME mapping file to process your data. However,
+the QIIME mapping file, in general, doesn't have all the EBI/ENA fields. Thus, you will need to
+update your templates directly in the system to add/modify these fields.
```
Comment (josenavas) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: Can this last sentence be rewritten? As it is right now it looks like you can change those templates directly through an interface in the system, which is not correct. Probably something along these lines:
`Thus, you will need to update your templates by downloading the system templates to add/modify these fields`

**Comment by josenavas** [comment]
Will try to download and check the rendering later today.

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst (hunk 1 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: Done

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst (hunk 2 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: Good point. Added a bit more info.

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Qiita allows users to deposit their study, sample, experiment and sequence data to the
+`European Nucleotide Archive (ENA) <https://www.ebi.ac.uk/ena>`__, which is a permanent repository
+part of the `European Bioinformatics Institute (EBI) <https://www.ebi.ac.uk/>`__. Submitting to
+this repository will provide you with a unique identifier for your study, which is generally a
+requirement for publication.
+
+EBI/ENA requires a given set of column fields to describe your samples and experiments, for more
+information visit :doc:`prepare-templates` and pay most attention to EBI required fields,
+without these Qiita Admins will not be able to submit. If you want to submit your data or need
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: Suggest bolding Qiita Admins i.e. `**Qiita Admins**`.

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+help send an email to `qiita.help@gmail.com <qiita.help@gmail.com>`__.
+
+Note that this kind of submissions are time intensive and need full collaboration from the user.
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: intensive -> consuming

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+Thus, do not wait until the last minute to request help. In general, the best time to request a submission
+is when you are writing your paper. Remember that the data can be submitted to EBI and can be
+kept private and simply make public when the paper is accepted. Note that EBI/ENA takes up to 15 days to
+change the status from private to public, so consider this when submitting data and your manuscript.
+
+For convenience Qiita allows you to upload a QIIME mapping file to process your data. However,
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: Would you mind linking to qiime.org when QIIME is first mentioned here?

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+the QIIME mapping file, in general, doesn't have all the EBI/ENA fields. Thus, you will need to
+update your information files (sample or preparation) via the update option. To simplify this process,
+you can download the system generated files and add and/or modify these fields for each file.
```
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: add and/or -> add/or

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst (hunk 5 above)
Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: Would probably make sense to put this whole paragraph inside an alert, like here (but maybe highlighted in red as an error): 
http://qiita.microbio.me/static/doc/html/tutorials/analyze-data.html

**Comment by ElDeveloper** [comment]
:+1: aside from the minor comments.

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst (hunk 3 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: Done

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst (hunk 4 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: Done

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst (hunk 5 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: Done

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst (hunk 6 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: replace it.

File: qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst (hunk 5 above)
Comment (antgonza) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: However, after displaying it as an error I realized that it adds the header "Error", which is misleading, also tried warning and is also misleading. Thus, decided by note, which I liked.

Comment (ElDeveloper) [inline qiita_pet/support_files/doc/source/tutorials/ebi-submission.rst]: Sounds excellent, thanks for trying them out :D

**Pull request merged**